src/index.js

```diff
@@ -101,6 +101,16 @@ const rule = (properties, options) =>
           }
         }
 
+        if (context.fix && !disableFix) {
+          // only if a fix code is available from config
+          if (options.autofixFunc) {
+            // take the fix function from the config
+            // eslint-disable-next-line no-param-reassign
+            node.value = options.autofixFunc(node.value)
```

Comment:
I would allow for much more sophisticated value fixing, by providing:

- [ ] `root`
- [ ] `node`
- [ ] `validVar`, `validFunc` and `validKeyword`

Providing the **whole PostCSS API** enabling generation of `$variables` and letting them be prepended for use in this rule.

Reply:
Actually the value should be fixed within the autofixing function.